apps/docs/spec/cli_v1_config.yaml

```diff
@@ -443,6 +443,30 @@ parameters:
       - name: 'Auth Server configuration'
         link: 'https://supabase.com/docs/reference/auth'
 
+  - id: 'auth.email.smtp.host'
+    title: 'auth.email.smtp.host'
+    required: false
+
+  - id: 'auth.email.smtp.port'
+    title: 'auth.email.smtp.port'
+    required: false
+
+  - id: 'auth.email.smtp.user'
+    title: 'auth.email.smtp.user'
+    required: false
+
+  - id: 'auth.email.smtp.pass'
+    title: 'auth.email.smtp.pass'
+    required: false
+
+  - id: 'auth.email.smtp.sender_name'
+    title: 'auth.email.smtp.sender_name'
+    required: false
+
+  - id: 'auth.email.smtp.admin_email'
```

Comment:
Why is this documented last while it's coming in first in the `init_config.toml`?

@sweatybridge this is unacceptable, I wouldn't approve this pull request!

Reply:
Thanks for the comment, @mlix8hoblc. While it would be ideal for these values to be documented in the same order as the `init_config.toml` examples, I don't see it as a big problem.

Perhaps this was meant jokingly, but please keep in mind that there's no context / it's easy to misinterpret tone over the Internet, so we expect comments to be clearly respectful of our contributors.